CMakeLists.txt

```diff
@@ -14,11 +14,16 @@
 # OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 #
 
-# Note: In order for RPM packaging to work correctly version >= 3.8 is required
-cmake_minimum_required(VERSION 3.8.0)
+if(EXISTS "/etc/debian_version")
+  cmake_minimum_required(VERSION 3.7.2)
+else()
+  # KLUDGE: assumes RPM here
+  # For RPM packaging to work correctly version >= 3.8 is required
+  cmake_minimum_required(VERSION 3.8.0)
+endif()
```

Comment:
Seems like a reasonable enough hack :)

Reply:
Maybe better the other way around? Apart from RPMs I'm not sure there is any reason to need the later `cmake`. Certainly the only reason I put 3.7.2 was because that's the version currently in Debian, but it likely works fine with earlier versions.

Reply:
i.e. look for `/etc/redhat-release` or `/etc/centos-release`, and otherwise don't set the minimum required version (or set to e.g. 3.0.0)